saleor/graphql/account/mutations/base.py

```diff
@@ -337,6 +346,8 @@ def clean_input(cls, info, instance, data):
             )
             cleaned_input[BILLING_ADDRESS_FIELD] = billing_address
 
+        # DEPRECATED: We shoud remove this confition whe drop `send_password_email`
```

Comment:
Something's wrong with the grammar in this comment. Check this sentence in Grammarly.